.env.example

```
@@ -26,6 +26,7 @@ DRIVER_SERVICE_URL=http://driver-service:8082
 # --- AWS / SQS Configuration (Primary Event Bus) ---
 # [FINALIZED] RabbitMQ sections removed in favor of AWS SQS FIFO
 AWS_REGION=us-east-2
+export AWS_ACCOUNT_ID=<YOUR_AWS_ACCOUNT_ID>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_ACCOUNT_ID key should go before the AWS_REGION key

(UnorderedKey)